scss/_variables.scss

```diff
@@ -0,0 +1,29 @@
+$container-width: 860px;
+$medium-padding: 40px;
+
+$color-1: #eeeeef;
+$color-2: #ffffff;
+$color-3: black;
+$color-4: #384048;
+$color-5: #50c3af;
+$color-6: #787b80;
+$color-7: #d2d5da;
+
+$medium-weight: 500;
+$black-weight: 900;
+$light-weight: 300;
+
+$basic-font-stack: PTSans-Regular, AdobeInvisFont, MyriadPro-Regular;
+$heading-font-stack: PTSans-Bold, AdobeInvisFont, MyriadPro-Regular,
+  $basic-font-stack;
+
+@mixin to640 {
+  @media only screen and (max-width: 768px + 17px) {
```

Comment:
why `+ 17px`?

Reply:
I saw this trick on the internet ) And I thought that it is a good idea to consider additional - kind of padding for the content of the website. 
Deleted it to avoid the confusion.